install.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# By default, this script will install the latest version of run to $HOME/.local/bin.
+install_dir="$HOME/.local/bin"
+version="latest"
+
+# Get the actual version from the latest release
+if [ "$version" = "latest" ]; then
+    version=$(curl -s https://api.github.com/repos/amonks/run/releases/latest | grep tag_name | cut -d '"' -f 4)
+fi
+
+# Detect OS and Architecture
+os=""
+arch=$(uname -m)
+os_type=$(uname -s)
```

Comment:
I think `uname` and `arch` return the right values when called with no flags, getting rid of the need for the switch/case:

    run_$(uname)_$(arch).tar.gz